com.unity.render-pipelines.universal/Editor/ShaderGraph/AssetCallbacks/CreateSpriteLitShaderGraph.cs

```diff
@@ -5,7 +5,7 @@ namespace UnityEditor.Rendering.Universal.ShaderGraph
 {
     static class CreateSpriteLitShaderGraph
     {
-        [MenuItem("Assets/Create/Shader/Universal Render Pipeline/Sprite Lit Shader Graph", false, 300)]
+        [MenuItem("Assets/Create/Shader Graph/URP/Sprite Lit Shader Graph", false, 1084)]
```

Comment:
I am not sure to get why some priorities are hard coded and some other are registered in const

Reply:
This I miss it. Thanks for  getting it